CONTRIBUTING.md

```diff
@@ -31,7 +31,38 @@ To setup a local development environment we recommend reading [Building Grafana
 
 - Add tests relevant to the fixed bug or new feature.
 
-- Follow [PR and commit messages guidelines](#PR-and-commit-messages-guidelines)
+### High level checks
+
+- [ ] The pull request adds value and the impact of the change is in line with [Backend](https://github.com/grafana/grafana/tree/master/pkg) or [Frontend](https://github.com/grafana/grafana/tree/master/style_guides).
+- [ ] The pull request works the way it says it should do.
+- [ ] The pull request does not increase the Angular code base.
```

Comment:
Shouldn't this be in the frontend specific checks section

Reply:
Good point, will move it!